avm/ptn/alz/empty/tests/e2e/max/main.test.bicep

```diff
@@ -0,0 +1,298 @@
+              exists: false
+            }
+          ]
+        }
+        then: {
+          effect: '[[parameters(\'effect\')]'
+        }
+      }
+    }
+  }
+]
+
+var alzCustomPolicySetDefsJson = [
+  loadJsonContent('lib/policy_set_definitions/Audit-TrustedLaunch.alz_policy_set_definition.json')
+  loadJsonContent('lib/policy_set_definitions/Deploy-MDFC-Config_20240319.alz_policy_set_definition.json')
+  {
+    name: 'custom-tags-policy-set-definition-1'
+    properties: {
+      description: 'Custom tags policy set definition'
+      displayName: 'Custom tags policy set definition'
+      policyType: 'Custom'
+      metadata: {
+        category: 'Tags'
+        version: '1.0.0'
+      }
+      parameters: {
+        tagName: {
+          type: 'String'
+          metadata: {
+            description: 'Name of the tag, such as costCenter'
+          }
+        }
+        effect: {
+          type: 'String'
+          metadata: {
+            displayName: 'Effect'
+            description: 'Audit or Disabled the execution of the Policy'
+          }
+          allowedValues: [
+            'Audit'
+            'Disabled'
+          ]
+          defaultValue: 'Audit'
+        }
+      }
+      policyDefinitions: [
+        {
+          policyDefinitionId: '{customPolicyDefinitionScopeId}/providers/Microsoft.Authorization/policyDefinitions/custom-tags-policy-1'
+          policyDefinitionReferenceId: 'custom-tags-policy-1'
+          parameters: {
+            tagName: {
+              value: '[[[parameters(\'tagName\')]'
+            }
+            effect: {
+              value: '[[[parameters(\'effect\')]'
+            }
+          }
+        }
+      ]
+    }
+  }
+]
+
+var managementGroupCustomRoleDefinitions = [
+  for policy in alzCustomPolicyDefsJson: {
+    name: policy.name
+    properties: {
+      description: policy.properties.description
+      displayName: policy.properties.displayName
+      metadata: policy.properties.metadata
+      mode: policy.properties.mode
+      parameters: policy.properties.parameters
+      policyType: policy.properties.policyType
+      policyRule: policy.properties.policyRule
+    }
+  }
+]
+
+var managementGroupCustomPolicySetDefinitions = [
+  for policy in alzCustomPolicySetDefsJson: {
+    name: policy.name
+    properties: {
+      description: policy.properties.description
+      displayName: policy.properties.displayName
+      metadata: policy.properties.metadata
+      parameters: policy.properties.parameters
+      policyType: policy.properties.policyType
+      version: policy.properties.?version
+      policyDefinitions: policy.properties.policyDefinitions
+    }
+  }
+]
+
+var managementGroupRoleAssignments = [
+  {
+    principalId: 'd543c6f0-89ce-4d51-a9fd-5b61986278a9'
+    roleDefinitionIdOrName: 'Security Reader'
+  }
+  {
+    principalId: deployer().objectId
+    roleDefinitionIdOrName: 'Reader'
+  }
+  {
+    principalId: 'd543c6f0-89ce-4d51-a9fd-5b61986278a9'
```

Comment:
Same comment as before

Reply:
correct have adjusted to deployer().objectId